### DISCUSSION TITLE
이모지 버그 이슈

### DISCUSSION THREAD

**Discussion by zooneon** [dnd-side-project/dnd-5th-3-backend#130] [질문] [answered]
성효님 저번에 발생했던 이모지 버그 이슈(DB collation 관련)
어떻게 해결하셨는지 공유해주실 수 있으신가요?

**Comment by KangSeongHyo** [comment, marked as answer]
아 그부분은 Mysql 문제였는데 
![image](https://user-images.githubusercontent.com/36832676/140628055-ff019f5e-6a02-4b91-8077-5fc1a9848e09.png)
collation이 utf8mb4_unicode_ci 인 경우에 이모지에 내용에 관계없이 동일하게 인식하는 부분이 있었는데
  ->  ut8mb4_0900_ai_ci 변경하니까 각각 독립적으로 인식해서 요걸로 해결했습니다.
